qnx/Makefile

```diff
@@ -0,0 +1,2 @@
+LIST=CPU
+include recurse.mk
```

Comment:
Can we move the QNX folder into `lvgl/env_support`?

Reply:
Sure